simple/sample/main_dc_output/schema.mcf

```diff
@@ -0,0 +1,46 @@
+Node: dcid:custom/g/group_1
+typeOf: StatVarGroup
+name: "Parent Group"
+specializationOf: dc/g/Root
+
+Node: dcid:custom/g/group_2
+typeOf: StatVarGroup
+name: "Child Group 1"
+specializationOf: custom/g/group_1
+
+Node: dcid:custom/g/group_3
+typeOf: StatVarGroup
+name: "Child Group 2"
+specializationOf: custom/g/group_1
+
+Node: dcid:custom/g/Root
+typeOf: StatVarGroup
+name: "Custom Variables"
+specializationOf: dc/g/Root
+
+Node: dcid:var1
+typeOf: StatisticalVariable
+name: "Good var1 name"
+description: "Good var1 description"
+memberOf: custom/g/group_2
+
+Node: dcid:var2
+typeOf: StatisticalVariable
+name: "Good var2 name"
+memberOf: custom/g/group_3
+
```

Comment:
This will fail validation.  SV needs properties like populationType, measuredProperty, statType?

Reply:
Added populationType and measuredProperty. PLMK if they look correct.

As for statType - should it be something users need to specify in a config? And what should be the default?

Reply:
https://github.com/datacommonsorg/data/blob/master/docs/representing_statistics.md#statisticalvariable 

`measuredValue` is the default

Reply:
Perfect. Filed b/322909728